napari/components/viewer_model.py

```diff
@@ -1000,6 +1000,69 @@ def add_vectors(
         self.add_layer(layer)
         return layer
 
+    def _add_layer_data(
```

Comment:
This is totally a nitpick, but I think the name `_add_layer_data()` is confusing. That being said, I'm not sure what is better. Maybe `_add_layer_from_data()` (too long?)?

Reply:
yeah, I meant to mention that in the PR.  I'm not in love with it either and definitely open to options.  I don't mind `_add_layer_from_data` ... maybe we can one more person to weigh in and I'll go with the consensus.  The other question is: does everyone agree this should be a _private method?

Reply:
I'm :+1: on `_add_layer_from_data`, and yes, let's keep it private for now, in case we want to change it as we use it more.